### ISSUE TITLE
Defaults

### ISSUE THREAD

**Issue by alanef** [jamesckemp/WordPress-Settings-Framework#17]
Thanks for your code, it is nice, I came across a small issue in using it

In your example settings you use the term 'std'

and also in the framework

protected $setting_defaults = array(
            'id'           => 'default_field',
            'title'        => 'Default Field',
            'desc'        => '',
            'std'          => '',
            'type'         => 'text',
            'placeholder' => '',
            'choices'     => array(),
            'class'       => '',
            'subfields'   => array()
        );

But this was not working - so I examined further the code is actually keyed on the term 'default' and using 'default' works so basically a documentation / example issue

**Comment by BriKKou** [comment]
Thanks @alanef that helps me a lot!
And thanks to the contributors for this very useful tool.

**Comment by jamesckemp** [comment]
Yep, sorry - I changed that when I took over. I'll update the docs.
